.github/workflows/ci.yml

```diff
@@ -20,24 +24,52 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.17.x
-      - name: Retrieve get module name
-        run: |
-          echo "::set-output name=Module::$(go list -m)"
-        id: module-name
-      - name: Verify builds with xk6
+          go-version: 1.20.x
+      - name: Check build
         run: |
-          go install go.k6.io/xk6/cmd/xk6@latest
-          xk6 build --with ${{ steps.module-name.outputs.Module }}=.
+          go version
+          pwd && ls -l
+          
+          go install go.k6.io/xk6/cmd/xk6@master
```

Comment:
This is consistent with the other extensions. If the use of `master` becomes an issue or we wish to switch to a Docker-based build, we can address it at that time.

Reply:
I appreciate the consistency, but using `master` is unsafe. xk6 is generally stable, but occasionally we might push an incompatible change. So unless we need to rely on bleeding edge features, it would be safer to use `latest` here.

Using the xk6 Docker image would also simplify and standardize CI workflows, so I don't see a reason to not use it.

But OK, we can address this later.